### PR TITLE
Record metadata

### PR DESCRIPTION
**Pull request by cube2222**

### CODE REVIEW

File: execution/record.go [hunk 1]
```diff
@@ -75,9 +132,26 @@ func (r *Record) Equal(other *Record) bool {
...
+	}
+
 	return true
 }
 
+func (r *Record) IsUndo() bool {
+	return r.metadata.undo
+}
+
+func (r *Record) EventTimeField() octosql.Value {
```
Comment (woojtek) [inline execution/record.go]: Is this used somewhere? Why not using it in line 73

File: execution/record.go [hunk 2]
```diff
@@ -43,6 +90,16 @@ func (r *Record) Fields() []Field {
 			Name: fieldName,
 		})
 	}
+	if !r.metadata.eventTime.IsZero() {
+		fields = append(fields, Field{
+			Name: octosql.NewVariableName("sys.event_time"),
+		})
+	}
+	if r.metadata.undo {
```
Comment (woojtek) [inline execution/record.go]: Maybe isUndo() here?

File: execution/record.go (hunk 1 above)
Comment (cube2222) [reply execution/record.go]: Fixed

File: execution/record.go (hunk 2 above)
Comment (cube2222) [reply execution/record.go]: Fixed

**Pull request merged**